### ISSUE TITLE
Training converges but only to a minimum value of 30 for scaled_flow_R_L6_loss

### ISSUE THREAD

**Issue by adhara123007** [twhui/LiteFlowNet#38]
Dear Sir,

Thank you for being actively replying to our queries. 
I am fine tuning the network based on the weight "liteflownet". My dataset is different than the datasets used for your study. Hence the need for fine tuning. I am concentrating only on loss 6. Other losses have weight 0.

[Liteflownet_training.log](https://github.com/twhui/LiteFlowNet/files/3144999/Liteflownet_training.log)

The loss starts decreasing but it stops improving beyond 30. I am attaching my log file. Do you expect the loss to go further down?

I0504 23:23:12.889382  2649 solver.cpp:245]     Train net output #0: scaled_flow_D1_L2_loss = 12066.4
I0504 23:23:12.889410  2649 solver.cpp:245]     Train net output #1: scaled_flow_D1_L3_loss = 2873.06
I0504 23:23:12.889416  2649 solver.cpp:245]     Train net output #2: scaled_flow_D1_L4_loss = 654.38
I0504 23:23:12.889422  2649 solver.cpp:245]     Train net output #3: scaled_flow_D1_L5_loss = 141.828
I0504 23:23:12.889436  2649 solver.cpp:245]     Train net output #4: scaled_flow_D1_L6_loss = 33.7398 (* 0.32 = 10.7967 loss)
I0504 23:23:12.889442  2649 solver.cpp:245]     Train net output #5: scaled_flow_D2_L2_loss = 12453.9
I0504 23:23:12.889447  2649 solver.cpp:245]     Train net output #6: scaled_flow_D2_L3_loss = 2963.34
I0504 23:23:12.889453  2649 solver.cpp:245]     Train net output #7: scaled_flow_D2_L4_loss = 663.826
I0504 23:23:12.889459  2649 solver.cpp:245]     Train net output #8: scaled_flow_D2_L5_loss = 142.224
I0504 23:23:12.889483  2649 solver.cpp:245]     Train net output #9: scaled_flow_D2_L6_loss = 33.0278 (* 0.32 = 10.5689 loss)
I0504 23:23:12.889489  2649 solver.cpp:245]     Train net output #10: scaled_flow_R_L2_loss = 12529.9
I0504 23:23:12.889497  2649 solver.cpp:245]     Train net output #11: scaled_flow_R_L3_loss = 3000.42
I0504 23:23:12.889502  2649 solver.cpp:245]     Train net output #12: scaled_flow_R_L4_loss = 689.57
I0504 23:23:12.889508  2649 solver.cpp:245]     Train net output #13: scaled_flow_R_L5_loss = 152.621
I0504 23:23:12.889531  2649 solver.cpp:245]     Train net output #14: scaled_flow_R_L6_loss = 32.6008 (* 1 = 32.6008 loss)

**Comment by twhui** [comment]
If you use your own training set, it is rather difficult for me to comment the performance as I have no idea about what is your training set.
